src/main/java/ch/uzh/ifi/hase/soprafs21/service/JwtTokenUtil.java

```diff
@@ -0,0 +1,71 @@
+package ch.uzh.ifi.hase.soprafs21.service;
+
+import ch.uzh.ifi.hase.soprafs21.security.config.SecurityConstants;
+import io.jsonwebtoken.*;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Service;
+import org.springframework.web.server.ResponseStatusException;
+
+import java.util.Date;
+
+@Service
```

Comment:
Does it have to be as Service, since this is just an utility class?

Reply:
I made it a component, so spring manage its lifecycle